base_app/README.md

```diff
@@ -1,62 +1,12 @@
-# NgSimpleUi
+# Base_app
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 7.1.1.
-
-The project was updated to `Angular 13.3` in several phases using the instructions in the [Angular Update Guide](https://update.angular.io/?l=3&v=7.2-13.0)
-with the following settings:
-
-**Angular Versions**  
-From: `7.2`, To: `8.0`  
-From: `7.2`, To: `9.0`  
-From: `7.2`, To: `10.0`  
-From: `7.2`, To: `11.0`  
-From: `7.2`, To: `13.0`
-
-**App Complexity** `Advanced`
-
-**Other Dependencies**
-- [ ] I use ngUpgrade to combine AngularJS & Angular
-- [x] I use Angular Material
-- [ ] I use Windows
-
-## Build
-
-Run `ng build-client-dev` to build a version of the `base-app` that contains
- `.map.js` files that allow direct debugging of `typescript` inside a
-browser debugger.
-
-Run `ng build-client-prod` to build an optimized version without the `.map` files
-that's intended for distribution.
-
-The build artifacts are stored in the `dist/` directory.
```

Comment:
These comments are useful because they describe how to do a build that creates .map files, which facilitate UI debugging.

Reply:
good point, not sure why I deleted that. Just added it back